langstream-core/src/main/java/ai/langstream/impl/common/BasicClusterRuntime.java

```diff
@@ -342,7 +342,7 @@ private Topic buildImplicitTopicForDeadletterQueue(
             StreamingClusterRuntime streamingClusterRuntime,
             ExecutionPlan physicalApplicationInstance) {
         // connecting two agents requires an intermediate topic
-        String name = connection.topicName() + "-deadletter";
+        String name = inputTopicDefinition.getName() + "-deadletter";
```

Comment:
We use the TopicDefinition name instead of the topic name. Is it OK?

Reply:
if all the tests are passing then I guess that it is okay